react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+const rootEl = document.querySelector('#root');
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+ReactDom.render(
+  CustomButton(), rootEl
```

Comment:
In react, we do not call our own components. Instead, create a react element like this:
```jsx
ReactDOM.render(
  <CustomButton />,
  rootEl
);
```